web/api/web_api_v1.c

```diff
@@ -208,13 +206,29 @@ inline int web_client_api_request_v1_alarms(RRDHOST *host, struct web_client *w,
         else if(!strcmp(value, "active")) all = 0;
     }
 
+    return all;
+}
+
+inline int web_client_api_request_v1_alarms(RRDHOST *host, struct web_client *w, char *url) {
+    int all = web_client_api_request_v1_alarms_select(url, 0);
+
     buffer_flush(w->response.data);
     w->response.data->contenttype = CT_APPLICATION_JSON;
     health_alarms2json(host, w->response.data, all);
     buffer_no_cacheable(w->response.data);
     return HTTP_RESP_OK;
 }
 
+inline int web_client_api_request_v1_alarms_values(RRDHOST *host, struct web_client *w, char *url) {
+    int all = web_client_api_request_v1_alarms_select(url, 1);
```

Comment:
I think this is confusing. 

I would expect `/alarms` and `/alarms_values` to behave the same i.e. have the same default for `all` if nothing is given

Reply:
I changed, because I imagined that our necessity would be to have all, so I changed in case the developer  forgot to add the correct parameter, we would not lose any information.